File: 프로그래머스/서울에서김서방찾기.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

string solution(vector<string> seoul) {
	string answer = "";
	string str1 = "김서방은 ";
	string str2 = "에 있다.";
	for (int i = 0; i < seoul.size(); i++) {
		if (seoul[i] == "Kim") {
			return str1 + to_string(i) + str2;
		}
	}
	return answer;
}

int main() {
	vector<string> str = { "Jane", "Kim" };
	cout << solution(str);
}

File: 프로그래머스/서울에서김서방찾기.py
def solution(seoul):
    answer = ''
    str1 = "김서방은 "
    str2 = "에 있다"
    for i in range(len(seoul)):
        if seoul[i]=="Kim":
            return str1+str(i)+str2
    return answer


tseoul = ["Jane", "Kim"]
print(solution(tseoul))
